tests/sentry/integrations/slack/test_notifications.py

```diff
@@ -355,3 +370,42 @@ def test_deploy(self, mock_func):
             attachment["footer"]
             == "<http://testserver/settings/account/notifications/?referrer=ReleaseActivitySlack|Notification Settings>"
         )
+
+    # @responses.activate
```

Comment:
Use `@pytest.mark.skip(reason=...)` here instead of commenting out the test. Commented out code is harder to maintain.